site/source/docs/compiling/Deploying-Pages.rst

```diff
@@ -0,0 +1,162 @@
+.. _Deploying-Pages:
+
+===================================
+Deploying Emscripten Compiled Pages
+===================================
+
+Emscripten compiled output can either be run directly in a JS shell from command line, or hosted on a web page. When hosting asm.js and WebAssembly compiled pages as .html for browsers to execute, Emscripten provides a default `HTML shell file` that serves as a launcher to run the code, simplified to get started with development. However when getting ready to release and host the content on a web site, a number of extra features and customizations are likely in order to polish the visitor experience. This guide highlights things to pay attention to when deploying sites to a CDN.
+
+Build Files and Custom Shell
+============================
+
+Emscripten build output consists of two essential parts: 1) the low level compiled code module and 2) the JavaScript runtime to interact with it. If targeting WebAssembly, the compiled code is stored in a file ``out.wasm`` and the runtime lives in a file ``out.js``. When targeting asm.js there exists an additional binary file ``out.mem`` that contains the static memory section of the compiled code. This part is embedded in the ``out.wasm`` file when targeting WebAssembly.
```

Comment:
maybe add a comment that "out" is the name of the output in these example names (so people don't think it's special)